ui/raidboss/data/02-arr/raid/t1.js

```diff
@@ -119,4 +119,71 @@
       },
     },
   ],
+  timelineReplace: [
+    {
+      'locale': 'de',
+      'replaceSync': {
+        'Ads': 'Abwehrsystem',
+        'Caduceus': 'Caduceus',
+        'Dark Matter Slime': 'Dunkelmaterien-Schleim',
+        'The Allagan megastructure': 'Allagische Superstruktur',
+      },
+      'replaceText': {
+      },
+      '~effectNames': {
+      },
```

Comment:
Giftpollen 
it's debuff of spuck on Caduceus

Reply:
as it is not used in either the txt or js file, its not needed here. 
I already asked @quisquous and the conclusion was to only include needed elements in the replacement section

Reply:
Ok, sorry. Same for other.

Reply:
dont worry :)